elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -22,25 +30,26 @@ $LOCAL-VARIABLES: (
     height: 100%;
   }
 
- //updated saturated styles
-  @at-root #{&}([pfe-variant="earth"][on="saturated"]) .container {
-    background-color: pfe-var(surface--lightest);
-    color: pfe-var(text);
-  }
-  @at-root #{&}([pfe-variant="earth"][on="saturated"]) ::slotted(*) {
-    color: pfe-var(text);
-}
-
   .container {
     margin: 0;
     width: 100%;
-    padding: pfe-local(Padding, $region: panel, $fallback: pfe-var(container-spacer) 0);
-
+    background-color: pfe-local(BackgroundColor, $region: panel);
     border-top:    pfe-local(BorderTop, 0, $region: panel);
     border-right:  pfe-local(BorderRight, 0, $region: panel);
     border-bottom: pfe-local(BorderBottom, 0, $region: panel);
     border-left:   pfe-local(BorderLeft, 0, $region: panel);
 
+    padding: pfe-local(Padding, $region: panel); //16px
+    @media screen and (min-width: pfe-breakpoint(md) ) {
```

```suggestion
    @media screen and (min-width: pfe-breakpoint(md) ) {
```

Did the pfe-breakpoint not require interpolation to work?  I thought it did!  That's cool.

---

I too was surprised!